style.css

```diff
@@ -0,0 +1,67 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+#main{
+    background-color: rgba(211, 211, 211, 0.5);
+    width: 90%;
+    height: fit-content;
+    margin:20px auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 15px;
+    
+}
+#title-description{
+    font-family: sans-serif;
```

Comment:
I notice that the "font-family" is the same for all tags that have this property applied.
I suggest making a class attribute to all those tags and putting the "font-family" property inside this class selector in the style.css file

Reply:
- [OPTIONAL] Nice one @Abdessittir , but try as much as you can to include `WHY` the student is asked to make use of a suggestion as this would make it clearer for the student to know why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where more information could be got for  more clarification.